build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -69,21 +87,69 @@ abstract class BuildCommitDistribution @Inject internal constructor(
         val rootProjectDir = project.repoRoot().asFile.absolutePath
         val commit = commitBaseline.map { it.substring(it.lastIndexOf('-') + 1) }
         val checkoutDir = RemoteProject.checkout(fsOps, execOps, rootProjectDir, commit.get(), temporaryDir)
+
+        fsOps.delete { delete(commitDistributionHome.get().asFile) }
```

Comment:
Do we also need to delete the TAPI JAR?

Reply:
Well, `commitDistributionHome` is a directory so we clean it up. For TAPI jar, it's overwritten anyway.